tests/conftest.py

```diff
@@ -58,6 +58,11 @@ def test_data_dir(base_test_dir):
 def workflows_config_dir(base_test_dir):
     return base_test_dir.parent / "nmdc_automation/config/workflows"
 
+@fixture(scope="session")
+def import_config_dir(base_test_dir):
+    return base_test_dir.parent / "configs"
```

Comment:
I have been keeping .yaml configurations for workflows in `nmdc_automation/config/workflows` and I think perhaps it would be better to keep import.yaml in a similar location such as `nmdc_automation/config/imports`

Reply:
I would like that to be outside of the scope of this PR, based on how the testing is currently set up if we use the default production config it won't be able to test all the needed functions so I explicitly left things as they are for now.